rocky/reports/views/base.py

```diff
@@ -71,7 +74,13 @@ def setup(self, request, *args, **kwargs):
         self.selected_report_types = request.GET.getlist("report_type", [])
 
     def get_oois(self) -> List[OOI]:
-        return [self.get_single_ooi(ooi_id) for ooi_id in self.selected_oois]
+        oois = []
+        for ooi_id in self.selected_oois:
+            try:
+                oois.append(self.get_single_ooi(ooi_id))
+            except Exception:
+                logger.warning("No data could be found for OOI ", ooi_id)
```

Comment:
Missing a `%s` specifier in the format string.
```suggestion
                logger.warning("No data could be found for '%s'", ooi_id)
```